app/lib/katello/util/cdn_var_substitutor.rb

```diff
@@ -90,21 +90,38 @@ def substituable?(path)
         path.include?("$")
       end
 
-      def valid_substitutions?(contnent_url, substitutions)
-        real_path = gsub_vars(contnent_url, substitutions)
+      def valid_substitutions!(content, substitutions)
```

Comment:
That being said, seems like this annoyance is preventing you guys from looking further so I 'll just change.